src/App.tsx

```diff
@@ -1,8 +1,28 @@
 import './styles/App.css';
 import Header from './ui/Header';
+import * as React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import YourRecipe from './pages/home';
+import Recipes from './pages/recipes';
+import About from './pages/about';
+import Admin from './pages/admin';
+import { Nav } from './ui/Nav';
 
 function App() {
-  return <Header></Header>;
+  return (
+    <Router basename="/oslo-kokebook/">
+      <div>
+        <Header />
+        <Nav />
```

Comment:
We may want these components to live on each page if we end up wanting more flexibility per page, but for now we can leave them here.